src/main/java/ca/ciccc/BankAccount.java

```diff
@@ -141,6 +170,17 @@ public boolean transfer(double amount, BankAccount other) {
     public String toString() {
         // TODO 1: Your code goes here.
 
-        return "";
+        String statement;
+        DecimalFormat money = new DecimalFormat("#.00");
+
+        if (balance >= 0){
+            statement = String.format(name + ", $" + money.format(balance));
```

Comment:
[FYI] You can also use `%.2f` for convert a double-type value into a string value. If you do it, you don't have to use `DecimalFormat`!

```suggestion
            statement = String.format("%s, $ %.2f", name, balance);
```